public/pages/Correlations/containers/CorrelationsContainer.tsx

```diff
@@ -287,7 +300,10 @@ export class Correlations extends React.Component<CorrelationsProps, Correlation
       <div style={{ backgroundColor: '#535353', color: '#ffffff', padding: '8px' }}>
         <EuiFlexGroup alignItems="center">
           <EuiFlexItem grow={false}>
-            <EuiBadge color={getSeverityColor(detectionRule.severity)}>
+            <EuiBadge
+              style={{ color: getSeverityColor(detectionRule.severity).text }}
+              color={getSeverityColor(detectionRule.severity).background}
```

Comment:
Nitpick: It might be slightly more efficient to create a variable to store the output of `getSeverityColor` instead of executing the function multiple times here.